source/features/pr-first-commit-title.tsx

```diff
@@ -37,6 +38,14 @@ function getFirstCommit(): {title: string; body: string | undefined} {
 }
 
 async function init(): Promise<void | false> {
+	const sessionResumeId = $('meta[name="session-resume-id"]')?.content;
+	const previousTitle = sessionResumeId && sessionStorage.getItem(`session-resume:${sessionResumeId}`);
+	if (previousTitle) {
```

Comment:
@pgimalac the code I had pasted worked, the only change it needed was that the resumeId is the target repo, not the current repo (`getRepo()`)

GitHub sets sessionStorage when the user leaves the page, and removes it on load, that's why it was hard to see.